modules/settings/templates/settings.php

```diff
@@ -6,22 +6,22 @@
                     <?php foreach ($context['modules'] as $module) : ?>
                         <?php if (!empty($context['module']['options_page']) && $context['module']->options->enabled === 'on') : ?>
                             <a
-                                    href="?page=<?php echo $context['slug']; ?>&module=<?php echo $context['module']->settings_slug; ?>"
+                                    href="?page=<?php echo esc_attr($context['slug']); ?>&module=<?php echo esc_attr($context['module']->settings_slug); ?>"
```

Comment:
Please, add all the URL in a `esc_url` function, removing the `esc_attr` function.

Reply:
updated